Microsoft.Toolkit.Uwp.SampleApp/SamplePages/TokenizingTextBox/TokenizingTextBoxXaml.bind

```diff
@@ -16,7 +16,7 @@
       <controls:TokenizingTextBox
           x:Name="TokenBox"
           PlaceholderText="Add Actions"
-          QueryIcon="{ex:SymbolIconSource Glyph=Setting}"
+          QueryIcon="{ex:SymbolIconSource Glyph=Setting, FontSize=16}"
```

Comment:
This is the part I'm not thrilled about, but not sure there's an easy workaround yet, unless I switch back to IconElement and copy it, but I'm not sure if that'll work either. I may just need to try that...

Reply:
I don't know if I can find an acceptable workaround ☹, the guidance might have to be that the developer must specify the FontSize here in order to get the control to look right if they use a QueryIcon. As if we switch back to `IconElement`, we can't easily duplicate the value to provide to each child, as the container refresh that happens when the `ItemsSource` is changed doesn't give the proper timing to remove the previous reference and avoid the duplicated tree error.

Reply:
Talked to the team, agree this is OK for now.